languages/go/exercises/concept/constants/.docs/after.md

```diff
@@ -0,0 +1,46 @@
+Constants in Go are simple, [unchanging values][const] created with the `const` keyword. Constants may be given an explicit type:
+
+```go
+    const hi string = "hi" // string
+```
+
+or an implicit, default type:
+
+```go
+    const hello = "hello" // string
+```
+
+when the variable needs a type. This gives constants more flexibility in Go's type system and allows them to work in a variety of contexts without triggering a compiler error:
+
+```go
+    sqrt2 := math.Sqrt(2) // 2 is an untyped numeric constant and does not need to be explicitly given the type float64 as required by the Sqrt method
```

Comment:
```suggestion
    const number2 = 2 // 2 is an untyped numeric constant and does not need to be explicitly given the type float64 as required by the Sqrt method
    sqrt2 := math.Sqrt(number2)
```

Reply:
Wasn't really aware of this 👍. Learned something today!

Note that if you use math.Sqrt(2), 2 is a basic literal, not really a constant (AFAIK). The rules might be the same but I'd make it more obvious with the suggested change.

Reply:
That example is straight out of Rob Pike's `constants` [blog post](https://blog.golang.org/constants#TOC_12.). I will expand it like you have.